user_tools/docs/user-tools-onprem.md

```diff
@@ -1,7 +1,7 @@
 # RAPIDS User Tools on OnPrem platform
 
 This is a guide for the RAPIDS tools for Apache Spark clusters that are provisioned manually (a.k.a onPrem). At the end of this guide, the user will be able to run the RAPIDS tools to analyze the clusters and the applications running OnPrem.
-
+Additionally, user can see cost savings and speedup recommendations for a comparable cluster on different cloud platform. Currently dataproc platform is supported.  
```

Comment:
Nit: new line starting from additionally (aka, add two blank spaces after OnPrem.)
```suggestion
RAPIDS tools to analyze the clusters and the applications running OnPrem. <Space><Space>
Additionally, user can see cost savings and speedup recommendations for a comparable cluster on different cloud platform. Currently "`dataproc`" platform is supported.  
```

Maybe we need more explanation of how that works and that by default it only calculates the speedups.